src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java

```diff
@@ -8,44 +8,44 @@
  */
 public class InputMethodName {
     void foo() {}
-    void Foo() {} //warn
-    void fOo() {} //warn
+    void Foo() {} // violation 'Method name 'Foo' must match pattern '.*[a-z][a-z0-9].*'.'
+    void fOo() {} // violation 'Method name 'fOo' must match pattern '.*[a-z][a-z0-9].*'.'
```

Comment:
original pattern was: `^[a-z][a-z0-9]\\w*$`

but had to modify it a little because it was causing error while comparing violation messages

Reply:
remove pattern.

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/9b97231e0ab7169577cce4aef9e5d03a3c19a06d/src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java#L11-L12